src/gui/advancedsettings.cpp

```diff
@@ -567,6 +573,22 @@ void AdvancedSettings::loadAdvancedSettings()
     m_spinBoxConnectionSpeed.setValue(session->connectionSpeed());
     addRow(CONNECTION_SPEED, (tr("Outgoing connections per second") + u' ' + makeLink(u"https://www.libtorrent.org/reference-Settings.html#connection_speed", u"(?)"))
             , &m_spinBoxConnectionSpeed);
+    // Socket send buffer size
+    m_spinBoxSocketSendBufferSize.setMinimum(0);
+    m_spinBoxSocketSendBufferSize.setMaximum(std::numeric_limits<int>::max());
+    m_spinBoxSocketSendBufferSize.setValue(session->socketSendBufferSize());
+    m_spinBoxSocketSendBufferSize.setSuffix(tr(" Bytes"));
+    m_spinBoxSocketSendBufferSize.setSpecialValueText(tr("System default"));
```

Comment:
I would set it as suffix so the value is visible.

Reply:
What about move it to the label just like in webui?

Reply:
>I would set it as suffix so the value is visible.
>
>>What about move it to the label just like in webui?

ps. I would prefer to leave it for a follow up PR since there are a few instances of it. Maybe you would like to handle it?

Reply:
>What about move it to the label just like in webui?

👍 since there are already several settings doing that but IMO it would be better to also see it in the spinbox, just my opinion.
>Maybe you would like to handle it?

I better not because i'm guessing you guys will want some abstarct funciton/template to use for all such settings.